### DISCUSSION TITLE
How to allow another user to only show censored calendar data ("busy")?

### DISCUSSION THREAD

**Discussion by kalsan** [Kozea/Radicale#1475] [Ideas]
Hi and thank you for this gem of software!

I'm planning to migrate an Exchange user to CalDAV with Radicale. One feature they use is that someone's private calendar is shared to other users but omitting the details, showing to the other user that they are "busy" but nothing else.

Can such a thing be achieved with Radicale?

Thanks in advance for your reply!
Best,
Kalsan

**Comment by pbiering** [comment]
unfortunatly, there is not even an easy to use way to share calendars at all (beside softlinking). Filtering on top of that would somehow require a major extension.

**Reply by kalsan** [reply]
Thank you for your quick and helpful reply! Would be amazing to have this feature at some point in the future :-)
